docs/install-1.7.md

```diff
@@ -123,7 +144,7 @@ export SC_SERVING_CERT=apiserver.pem
 export SC_SERVING_KEY=apiserver-key.pem
 ```
 
-## Get the appropriate tls ca/cert/key from kube
+## Get the Appropriate TLS CA, Certificate and Key from Kubernetes
```

Comment:
The heading levels might need to be adjusted, this was supposed to be a peer with the "generate the Cert Bundle" part, as it is an alternative and both should not be done at the same time.

Reply:
I put the two options under the same heading